core/cc/crash_handler.h

```diff
@@ -35,7 +35,12 @@ class CrashHandler {
       Handler;
   typedef std::function<void()> Unregister;
 
+#if TARGET_OS == GAPID_OS_ANDROID && TARGET_OS != GAPID_OS_LINUX
```

Comment:
This statement is weird 
`TARGET_OS == GAPID_OS_ANDROID implies TARGET_OS != GAPID_OS_LINUX`
Since TARGET_OS == GAPID_OS_ANDROID.

Reply:
b/145808085

Reply:
That cannot be right. Your `#if` is `a == 1 && a != 2`. If `a == 1` it can never also `a == 2`.

Reply:
I don't know what is the reason, I have not inspect it but it does not compile on Linux(at least on my machine). I can take a look at it after vacation.

Reply:
@yalcinmelihyasin the compilation error was due to a missing `#include "core/cc/target.h"`, this header is the one that properly defines TARGET_OS. I just pushed the fix.